katello/katello-backup

```diff
@@ -1,94 +1,113 @@
 #!/bin/bash
 
 usage(){
-  echo "Usage: katello-backup <backup directory>"
-  echo " eg: $ katello-backup /tmp/katello-backup/"
+  echo "Usage: katello-backup --directory <backup directory>"
+  echo " eg: $ katello-backup --directory /tmp/katello-backup/"
```

Comment:
why are you changing this from being a positional argument? specifying the directory is required, correct?